index.bs

```diff
@@ -810,6 +812,11 @@ consisting of the character U+00C5 LATIN CAPITAL LETTER A WITH RING ABOVE.</p>
   automatic position">auto</dfn>, which means the position is to depend on the <a lt="WebVTT cue
   text alignment">text alignment</a> of the cue.</p>
 
+  <p>For <a>WebVTT cues</a> that have a <a lt="WebVTT cue size">size</a> other than 100%, and a <a
+  lt="WebVTT cue text alignment">text alignment</a> of <a lt="WebVTT cue start alignment">start</a>
+  or <a lt="WebVTT cue end alignment">end</a>, authors must not use the default <a lt="WebVTT cue
+  automatic position">auto</a> <a lt="WebVTT cue position">position</a>.</p>
```

Comment:
Should this be restricted only to cases where "size" is less than 100%? Cause otherwise there is no positioning happening.

Also, would be good to explain why we require explicit positioning - maybe again in a note. Something about the "center" alignment of the text box may confuse captioners that have chosen "start/end" alignment of the text inside the box and feel that the box should be moved to the left or right. However, because of RTL and LTR and mixed directionality text, such automated positioning can lead to unexpected positioning, so therefore we require explicit positioning.